packages/office-ui-fabric-react/src/components/Check/Check.scss

```diff
@@ -9,74 +9,73 @@
 
 $checkBoxHeight: 24px;
 
-:global {
-  .ms-Check {
-    line-height: 1;
-    width: $checkBoxHeight;
-    height: $checkBoxHeight;
-    vertical-align: top;
-    position: relative;
-    @include user-select(none);
+.root {
+  line-height: 1;
+  width: $checkBoxHeight;
+  height: $checkBoxHeight;
+  vertical-align: top;
+  position: relative;
+  @include user-select(none);
 
-    &.is-checked {
-      .ms-Check-background {
-        &:before {
-          color: $ms-color-themePrimary;
-        }
-
-        &:after {
-          color: $ms-color-white;
-        }
+  &.isChecked {
```

Comment:
save flattening. It is error prone and i want to avoid any unnecessary regressions or changes. Our goal is ONLY to get css modules done. We can refine rules in separate passes, but doing too many things at once is asking for bugs.